docs/manual/developer/01_introduction.md

```diff
@@ -77,7 +77,23 @@ Those areas are going to be segmented by
 
 - Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
 - Area of effect: All PSPs that don’t have specific stakeholders, associated content and test scenarios.
-- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy paste, and also don’t change existing behavior for everybody (link to coding guidelines needed).
+- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy and paste, and also don’t change existing behavior for everybody.
+  See the [style guide](04_style_guide.md) for more details.
+
+##### Removal of Products
+Products that do not have any contributions for two years may be subject to removal.
+The following process shall be used:
+1. A pull request is created removing the product.
```

Comment:
```suggestion
1. The intention to remove the product is publicly communicated.
1. A pull request is created removing the product.
```

Creating a PR is quite difficult, so better to do it if we know that there will be no veto.